internal/armada/server.go

```diff
@@ -120,7 +132,7 @@ func Serve(config *configuration.ArmadaConfig, healthChecks *health.MultiChecker
 	grpcCommon.Listen(config.GrpcPort, grpcServer, wg)
 
 	return func() {
-		stopSubscription()
+		teardown()
```

Comment:
What grace period do we give for a server to shut down? In theory this could have added another 20 seconds to shut down time